### DISCUSSION TITLE
wrong German date translation

### DISCUSSION THREAD

**Discussion by cech12** [community/community#129415] [Mobile]
![Screenshot_20240622-103949.png](https://github.com/community/community/assets/16005054/8872bc23-29e9-4aec-aabd-0695843b2ea9)

The word "Vor" is doubled in the GitHub App (Android) (GitHub Mobile v1.165.0 (791))

https://github.com/users/cech12/projects/1

**Comment by aconsuegra** [comment]
Thanks a lot for reporting the issue! I have created an internal issue to track it, and we should have a fix ready in one of our upcoming releases.
